org-chart/src/App.test.tsx

```diff
@@ -1,9 +1,33 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { fireEvent, render, screen } from '@testing-library/react';
 import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+  const linkElement = screen.getByText(/THE AVATARS AND THEIR FRIENDS/i);
   expect(linkElement).toBeInTheDocument();
 });
+
+describe('Cards', () => {
+  it('renders first card', () => {
+    render(<App />);
+  
+    const cardElement = screen.getByText(/Kyoshi/i);
+    expect(cardElement).toBeInTheDocument();
+  })
+
+  it("doesn't render Roku ", () => {
+    render(<App />);
+    const cardElement = screen.queryByText(/Roku/i);
+    expect(cardElement).not.toBeInTheDocument();
+  }) 
+
+  it("renders new level after selecting a card", () => {
+    render(<App />);
+    const kyoshiCard = screen.getByText("Kyoshi");
+    fireEvent.click(kyoshiCard)
+    const rokuCard = screen.getByText("Roku");
+    expect(rokuCard).toBeInTheDocument();
+    
+  }) 
```

Comment:
```suggestion
  it("renders new level after selecting a card", () => {
    render(<App />);
    const kyoshiCard = screen.getByText("Kyoshi");
    fireEvent.click(kyoshiCard)
    const rokuCard = screen.getByText("Roku");
    
    expect(rokuCard).toBeInTheDocument();
  }) 
```

- Should remove blank lines
  - useful to check the `Files` section of your PR to check the diff and see what's being added
- Suggest splitting up test sections. Personally I don't add lines in between if the test is 3 lines or less. More than that, I usually split it by any mocks at the top, the test itself (render + interactions), followed by the results (expect statements). Some tests are more complicated and you could have multiple mock/test/result sections.